submissions/EnF1NeeT/HTML-CSS_popup/index.html

```diff
@@ -0,0 +1,155 @@
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/News.svg" alt="" class="icon"> News</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Storage.svg" alt="" class="icon"> Storage</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Saved.svg" alt="" class="icon"> Saved</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/recent.svg" alt="" class="icon"> Recent</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Apps.svg" alt="" class="icon"> Apps</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Locations.svg" alt="" class="icon"> Locations</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Post.svg" alt="" class="icon"> Posts</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/search.svg" alt="" class="icon"> Search</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/trash.svg" alt="" class="icon"> Trash</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/calendar.svg" alt="" class="icon"> Calendar</a></li>
+              </ul>
+              <input type="checkbox" id="menu-check-more" class="input-check popup-item">
+              <label for="menu-check-more" class="check-more popup-item">↓</label>
+              <div class="main-menu-more">
+                <ul class="popup-grid-more list">
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/manage_account.svg" alt="" class="icon"> Manage accounts</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/security.svg" alt="" class="icon"> Security</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/payments.svg" alt="" class="icon"> Payments</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/store.svg" alt="" class="icon"> Store</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Settings.svg" alt="" class="icon"> Settings</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/logout.svg" alt="" class="icon"> Logout</a></li>
+                </ul>
+              </div>
+            </div>
+          </li>
+          <li class="logo"><a href="./index.html" class="logo"><h1>Kottanspost</h1></a></li>
+          <li class="greeting"><h2>Hi, Kottan!</h2><img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt=""></li>
```

Comment:
Avoid morphing images, please fix the aspect ratio of the icon 
![image](https://user-images.githubusercontent.com/16196199/185799974-92d04de0-8dd2-4ff7-8d07-25304784ee6d.png)

Reply:
Fixed in the last commit